### ISSUE TITLE
[Firefox] "Proxy injection failed" while using Responsive Design Mode.

### ISSUE THREAD

**Issue by Hendriksie** [vuejs/vue-devtools#384]
**Firefox**: 54.0.1 (64-bit)
**Vue,js Devtools**: 3.1.6
**Vue.js**: 2.4.2

When using Firefox Responsive Design Mode, Vue.js Devtools is not initializing.
It's giving me the response: `Proxy injection failed`.

No error is given in the console, but 3 of our projects that's using Vue is giving me the same response.

One of these projects is using Sentry.io logging and we received an error:
```
TypeError: can't redefine non-configurable property "__VUE_DEVTOOLS_GLOBAL_HOOK__"
```
```
in installHook at line 66:3
```

Steps to reproduce:

1. Use Vue.js in your application
2. Have the official Vue.js Devtools installed in Firefox
3. Enable Responsive Design Mode
4. Open the Vue.js Devtools tab

**Comment by jsvini** [comment]
Same issue here: 

![2017-09-28_10-36-25](https://user-images.githubusercontent.com/4185886/30969498-f213bca4-a438-11e7-8c5f-6b5cf752411c.gif)

**Comment by AAhrit** [comment]
same here! mine doesn't even work in normal view

**Comment by andyedward** [comment]
same here! 
Vue 2.5.3
Firefox 57

**Comment by laphilosophia** [comment]
👍 same here

**Comment by rdpascua** [comment]
I'm having issue with firefox quantum too.

**Comment by yasare16** [comment]
**Chrome**: Version 63.0.3239.132
**Vue Tools**: Version 4.0.0
**VueJS**: 2.5.13

I've also noticed reloading with command + R(`⌘ + R`) causes the _Proxy Injection Failed_ error sometimes too. Reloading with the browser button gets it right again, though. Also important to note, my developer tools window is open each this happens. Happens more often as my project gets bigger.

This seems to be about Firefox but I thought I'd mention there are similar problems in Chrome

**Comment by tmladek** [comment]
Confirming, still exists in:

**Firefox:** 57.0.3 (64-bit)
**Vue.js devtools:** 4.1.3
**VueJS:** 2.5.13

**Comment by simplesmiler** [comment]
`TypeError: can't redefine non-configurable property "__VUE_DEVTOOLS_GLOBAL_HOOK__"` was addressed in 4.1.2, so if this issue still happens, it is unrelated to that error.

**Comment by Hendriksie** [comment]
I'm not getting the error anymore using:

**Firefox**: 62 (64-bit)
**Vue,js Devtools**: 4.1.4
**Vue.js**: 2.5.16

yay!

(what is the normal procedure to close issues like these, do I have to do it or a mod?)

**Comment by simplesmiler** [comment]
Good, glad to the the issue resolved!

**Comment by entepe85** [comment]
Unfortunately, this error still persists in Firefox Nightly. It always happens after the initial reload in responsive design mode (i.e. when the user agent changes) or after manually reloading in that mode.

**Firefox:** 69.0a1 (64-bit)
**Vue.js devtools:** 5.1.1
**Vue.js:** 2.6.10

Even a complete refresh of Firefox (thus also resetting customized about:config entries) didn't get me rid of this issue.

**Comment by entepe85** [comment]
This is the error message I get upon reloading the tab in Responsive Design Mode (from the browser debugging dev tools):

`Unchecked lastError value: Error: No matching message handler`

**Comment by entepe85** [comment]
Please reopen this issue as the problem persists across Firefox Nightly (69.0a1) and Developer Edition / Beta (69.0b1 as of today).

**Comment by zkriszti** [comment]
Restarting the browser seems to have solved this problem for me. (Firefox Developer Edition 69.0b1)

**Comment by entepe85** [comment]
Thanks, zkriszti. I tried restarting the browser but with no luck. After restarting the corresponding tab / app reloads in normal mode and upon entering RDM and the subsequent reloading of the app / page (due to touch emulation and user agent changes) I get the same error.

**Comment by entepe85** [comment]
Bug is fixed in Firefox Nightly 70.0a1 (2019-07-25). [Corresponding changes in mozilla-central](https://hg.mozilla.org/mozilla-central/log?rev=1562821)
